scripts/push-jar

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Script to build the jobs jar and push it to a specified s3 bucket.
+#
+# Usage:
+#
+#   scripts/push-jar <s3-folder-uri>
+#
+#   e.g.
+#
+#   scripts/push-jar s3://dpr-jars/dev
+
+set -e
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+s3_uri="$@"
+
+echo
+
+if [ -z "$s3_uri" ]
+then
+  echo "A destination s3 uri must be specified"
+  exit 1;
+fi
+
+echo "Building jar"
+echo
+
+cd $script_dir/../
+./gradlew clean shadowJar
+
+echo
+echo "Uploading jar to $s3_uri"
+echo
+aws s3 cp ./build/libs/digital-prison-reporting-jobs-*-all.jar $s3_uri/digital-prison-reporting-jobs-1.0-SNAPSHOT-all.jar
```

Comment:
do we need to specify the destination file name here? can we simplify like this?

aws s3 cp ./build/libs/digital-prison-reporting-jobs-*-all.jar $s3_uri/

Reply:
You're quite right. This is a hangover from a version I knocked up for myself a while back. Should've read the whole line.

Reply:
Fix pushed for this (dest filename has been removed) so resolving this.